src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -888,6 +888,18 @@ _do_spawn (const char *path, char **args, bson_error_t *error)
 }
 #else
 
+/**
+ * @brief Exit the calling process *immediately* without running any per-process
+ * cleanup handlers.
+ *
+ * @param status The exist status for the process.
+ */
+static void
+fast_exit (int status)
```

Comment:
Why `fast_exit ()` instead of directly calling `_exit ()`? Is it for platform-compatibility reasons?

Reply:
Initially yes, as `_exit()` is part of POSIX whereas `_Exit` is part of C99, and some platforms do not have `_Exit`. Since this segment of code is conditionally compiled on unix-like systems, calling `_exit()` directly will be fine.